js/stepper/shared/Tabs.svelte

```diff
@@ -0,0 +1,406 @@
+<script context="module" lang="ts">
```

Comment:
Is there a way to reduce the code dupe between `js/stepper/shared` and `js/tabs/shared`? Otherwise bugfixes would have to be made in two separate files

Reply:
Ah nice! Yes I was confused